ccmain/thresholder.cpp

```diff
@@ -312,7 +312,7 @@ void ImageThresholder::ThresholdRectToPix(Pix* src_pix,
       bool white_result = true;
       for (int ch = 0; ch < num_channels; ++ch) {
         int pixel = GET_DATA_BYTE(const_cast<void*>(
```

Comment:
I tried to keep this commit to just a(n admittedly long) list of reducing or removing reinterpret_cast, which should be used sparingly, but I'll leave any follow-up to whoever feels that urge.